cerealize/test/impl.cc

```diff
@@ -93,7 +96,7 @@ TEST(Cerealize, NamedJSON) {
 };
 
 TEST(Cerealize, BinarySerializesAndParses) {
-  FileSystem::CreateDirectory(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirectoryParameters::Silent);
+  FileSystem::CreateDir(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirParameters::Silent);
```

Comment:
Not ```Directory``` because Windows already ```#define```-s ```CreateDirectory``` into ```CreateDirectoryA```, and I don't want to conflict with it.

Not sure I got the ```creat/create``` comment.